components/TextFields/src/MDCTextField.m

```diff
@@ -86,7 +86,8 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     }
 
     [self commonMDCTextFieldInitialization];
-    _cursorColor = [aDecoder decodeObjectForKey:MDCTextFieldCursorColorKey];;
+    _cursorColor = [aDecoder decodeObjectForKey:MDCTextFieldCursorColorKey];
+    ;
```

Comment:
Nit: Remove extra semicolon.

Reply:
haha. Oh, clang-format. Thank you for replacing `;;` with `;\n;`.